app/src/main/res/values/strings.xml

```diff
@@ -140,7 +140,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="remember_last_directory">Remember last directory</string>
     <string name="syntax_highlighting">Syntax highlighting</string>
     <string name="highlighting_delay">Highlighting delay</string>
-    <string name="highlighting_delay_lower_value_more_battery_drain_description__appspecific">Delay in milliseconds for highlighting refresh. A lower value will start highlighting earlier, but will drain more battery and make a heavy load.</string>
+    <string name="highlighting_delay_lower_value_more_battery_drain_description__appspecific">Delay in milliseconds for highlighting refresh. A lower value will start highlighting earlier, but it will drain your battery, and may slow down the app on older devices.</string>
```

Comment:
", and" -> never both

Reply:
@gsantner Right, so how do i change that right now? Never really learned this "comment" system properly.

Reply:
comment system is just for chat.

depends how you want to - you can do git push from computer, or just edit in web editor. Easiest is the latter in this case, and just copy-paste the old string content

Reply:
Think I've worked it out. You have to send this as a review.

Reply:
It doesn't matter, I can send multiple as "one review" or just make several single review(s comments)

Reply:
ahhhh ok so pushing to the fork auto updates the pr gotit

Reply:
exactly

Reply:
yeah sorry about this mess, still learning the way with pr commenting. seems unintuitive to have to have two tabs open to do edit this tho (one to check lines & comments, one to edit)